vcd/resource_vcd_network_test.go

```diff
@@ -331,17 +518,34 @@ func runTest(def networkDef, t *testing.T) {
 				Config: configText,
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckVcdNetworkExists(networkName, &network),
-					testAccCheckVcdNetworkAttributes(networkName, &network),
+					checkNetWorkIpGroups(resourceDef, def, resourceVcdNetworkIPAddressHash),
```

Comment:
I am just worried about one thing with these sophisiticated checks:
When you have a check with hardcoded hash (like example below) - you are sure that the test catches hash change in our code in future. This is important because if we change the hash - all users will get "plan" updates even if nothing changes. Now when we use the hash function to get hash - we skip this check.
`resource.TestCheckResourceAttr("vcd_edgegateway.egw", "external_network.763048197.name", "test_external_network"),`

Reply:
I added a test that checks some hard coded hashes to make the hash function future proof